### ISSUE TITLE
threebot test framework [development_3bottest]

### ISSUE THREAD

**Issue by despiegk** [threefoldtech/home#420]
## goal

- run basic jumpscale tests on top of zero-os & tfgrid only using our tools
- run in parallel using the scheduler
- can define which branches to use

## principles

- use tfgrid to deploy threebot basecontainer
    - depends on 3bot builder: threefoldtech/jumpscaleX_core#262
- create jumpscale_test actor which 
    - uses scheduler
    - deploy required threebot containers (use scheduler event to wait till all containers there)
    - schedule different tests on the containers

## requirements

- [ ] have basic tests working using scheduler & myjobs & flists
    - [ ] builder for threebot for zos threefoldtech/jumpscaleX_core#262
    - [ ] test framework see threefoldtech/jumpscaleX_core#263

## remarks

- see #425 for future work on testing
- this story needs to be a very easy one, no new features, just use what we have

**Comment by AhmedHanafy725** [comment]
1- What are the basic tests needed here?
2- How will these tests be divided so it can be run in parallel?
3- will this testtool #481 be used? (as it does the same thing except creating the containers)
4- Will this framework be a part of [zeroCI](https://github.com/threefoldtech/zeroCI)?

**Comment by xmonader** [comment]
closed will be replaced with a new issue
